cortex_xdr_client/api/alerts_api.py

```diff
@@ -48,7 +50,8 @@ def get_alerts(self,
                               json_value=request_data)
         if response.ok:
             return GetAlertsResponse.parse_obj(response.json())
-        return None
+
+        return GetAllErrors.parse_obj(response.json())
```

Comment:
Type hinting is violated with this return statement :/

Reply:
We could raise an exception instead. And annotate the methods accordingly so the consumers are aware of this change. Wdyt? @lawiet47

Reply:
Yeah you're right it would be better implementing exceptions